qiskit_ionq/ionq_job.py

```diff
@@ -47,7 +47,31 @@
 from . import constants, exceptions
 
 
-def _build_counts(result, use_sampler=False, sampler_seed=None):
+def map_output(data, clbits, num_qubits):
+    """Map histogram according to measured bits"""
+
+    if not clbits:
+        return {}
+
+    mapped_output = {}
+
+    def get_bitvalue(bitstring, bit):
+        if bit is not None and 0 <= bit < len(bitstring):
+            return bitstring[bit]
+        return '0'
```

Comment:
Should we fail instead of returning 0 outside the bounds?

Reply:
Since at the end this comes from Qiskit measurement instructions I wonder if they validate this already 🤔 
Still failing could be a good idea, I'll take a look

Reply:
Ah gotcha. Not to worried about it in that case :+1: